### PR TITLE
Remove empty strings from the Census schemas

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?

Removes/fixes empty strings within the Census schemas in line with the update to eq-questionnaire-validator which enforces validation of empty string fields [(PR)](https://github.com/ONSdigital/eq-questionnaire-validator/pull/41).

### How to review

Check nothing is broken by this change.

### Checklist

- [ ] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

#### England

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=H&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/remove-empty-strings/schemas/en/census_household_gb_eng.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/remove-empty-strings/schemas/en/census_individual_gb_eng.json)

- [CCS](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/remove-empty-strings/schemas/en/ccs_household_gb_eng.json)

#### Northern Ireland

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/remove-empty-strings/schemas/en/census_household_gb_nir.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/remove-empty-strings/schemas/en/census_individual_gb_nir.json)

### CODE REVIEW

**Comment by berroar** [comment]
[Review the use of empty string values in schemas (S)](https://trello.com/c/YKDAmHxG/3743-review-the-use-of-empty-string-values-in-schemas-s)

File: source/jsonnet/england-wales/ccs_household.jsonnet [hunk 1]
```diff
@@ -164,7 +164,7 @@ function(region_code, census_month_year_date) {
       groups: [
         {
           id: 'accommodation-group',
-          title: '',
+          title: 'Accommodation group',
```
Comment (MebinAbraham) [inline source/jsonnet/england-wales/ccs_household.jsonnet:167]: If these aren't displayed, then we shouldn't define them.

Comment (berroar) [reply source/jsonnet/england-wales/ccs_household.jsonnet:167]: They are required properties at the moment, so I can't remove them unless I change validator, and can't leave them empty due to the new validation rules.

Comment (MebinAbraham) [reply source/jsonnet/england-wales/ccs_household.jsonnet:172]: We should consider whether we should remove keys that are not needed by runner. Obviously this is required in validator but not needed in runner, so defining keys that don't get used seems wrong.

**Comment by MebinAbraham** [comment]
If you haven't run this against your validator, I would point it at that so it can highlight any missing empty strings.

**Pull request merged**